internal/pkg/manifest/lb_fargate_manifest.go

```diff
@@ -82,8 +82,8 @@ func NewLoadBalancedFargateManifest(appName string, dockerfile string) *LBFargat
 	}
 }
 
-// Marshal serializes the manifest object into a YAML document.
-func (m *LBFargateManifest) Marshal() ([]byte, error) {
+// MarshalBinary serializes the manifest object into a binary YAML document.
```

Comment:
Why is it a binary YAML document? Thought this is an app manifest YAML document (`MarshalAppManifest`).

Reply:
Yes, it is. Maybe I should update the comment? When we write a file the input is a `[]byte` so this just the `[]byte` representation of the YAML doc.